baaas-decision-fleet-manager/src/main/resources/application.properties

```diff
@@ -71,6 +71,13 @@ quarkus.native.additional-build-args=-H:ReflectionConfigurationFiles=reflection-
 %dev.baaas.dfm.secrets-manager.aws.access-key-id=test-key
 %dev.baaas.dfm.secrets-manager.aws.secret-access-key=test-secret
 
+# Decision
+# -1 to disable, >= 0 forthe number of allowed decisions
+baaas.dfm.max.allowed.decisions=3
```

Comment:
By default lets check for the presence of an ENV variable or disable the functionality so something like:

```
baaas.dfm.max.allowed.decisions=${BAAAS_DFM_MAX_ALLOWED_DECISIONS:-1}
```

I suspect in all environments other than operate-first, we will not enforce any maximum allowed Decisions.